bindings/nodejs/lib/types/models/state.ts

```diff
@@ -13,9 +13,15 @@ export declare type BlockState =
 
 /**
  * The different states of a transaction.
+ * If 'pending', the transaction is not included yet.
+ * If 'accepted', the transaction is included.
+ * If 'confirmed' means transaction is included and its included block is confirmed.
```

Comment:
```suggestion
 * If 'confirmed', the transaction is included and its included block is confirmed.
```
For consistency, same for the 2 others